src/tape/ensemble.py

```diff
@@ -1160,14 +1163,13 @@ def from_parquet(
                     columns.append(col)
 
         # Read in the source parquet file(s)
-        self._source = dd.read_parquet(
-            source_file, index=self._id_col, columns=columns, split_row_groups=True
```

Comment:
just curious, why are we dropping the split_row_groups kwarg here and below?

Reply:
Yeah, a good question! This is just because in [`EnsembleFrame.from_parquet`](https://github.com/lincc-frameworks/tape/blob/main/src/tape/ensemble.py#L1180) we currently always have `split_row_groups=True`.

This was meant to match that we previously never let a user set to false, but of course we could open up the option (and then specify it here).